gemac/management.py

```diff
@@ -0,0 +1,165 @@
+from myhdl import block, Signal, intbv, always_seq, concat, now
+
+
+rx0, rx1, tx, flow, managementreg, ucast0, \
+    ucast1, addrtable0, addrtable1, addrfiltermode, reserved = range(11)
+
+
+class mdioData:
+    def __init__(self):
+        self.hostclk_count = Signal(intbv(0)[5:])  # Used to drive mdc
+        self.wrdata = Signal(intbv(0)[32:])
+        self.rddata = Signal(intbv(0)[16:])
+        self.rdindex = Signal(intbv(17, min=0, max=18))  # in index during rdop
+        self.wrindex = Signal(intbv(62, min=0, max=63))
+        self.wrdone = Signal(bool(0))
+        self.done = Signal(bool(0))
+
+"""
+Management Data - Input Output Interface.
+
+hostClk - Reference Clk for Management/Configuration Operations. >10MHz
+hostOpcode (2 Bits) - Define Operation for MDIO Interface. Bit 1 also
+    used as control signal for configuration data transfer.
+hostAddr (10 Bits) - Address of Register to be accessed. According to
+    Table 8.2, Page 27, Xilinx UG 144 doc.
+hostWriteData (32 bits) - Data write.
+hostReadData (32 bits) - Data read.
+hostMIIM_sel - Set by Host. When High MDIO interface Accessed,
+    Else Configuration registers.
+hostReq - Set by Host to Indicate ongoing MDIO transaction.
+hostMIIM_rdy - Set by MDIO Interface to indicate ready for new transaction.
+
+mdc - Management Clock: programmable frequency derived from host_clk.
+mdioIn - Input data signal from PHY for its configuration and
+    status.(TriStateBuffer Connected)
+mdioOut - Output data signal from PHY for its configuration and
+    status.(TriStateBuffer Connected)
+mdioTri - TriState Control for Signals - Low Indicating mdioOut to be
+    asserted to the MDIO bus.
+
+"""
+
```

Comment:
Should have more description what this ~~module~~ block does and why and with example usage.